test/e2e/basic_volume_io/basic_volume_io_test.go

```diff
@@ -0,0 +1,93 @@
+// JIRA: CAS-505
+// JIRA: CAS-506
```

Comment:
I think we should remove this?

Reply:
There was some discussion early on in which it was decided that we add this so for ease of tracking test results.